<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>331</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Control</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>301</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Lights</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="enable_lights_btn">
          <property name="text">
           <string>Enable Lights</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disable_lights_btn">
          <property name="text">
           <string>Disable Lights</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="enable_trackers_btn">
          <property name="text">
           <string>Enable Trackers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disable_trackers_btn">
          <property name="text">
           <string>Disable Trackers</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>301</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="enable_all_btn">
          <property name="text">
           <string>Enable All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disable_all_btn">
          <property name="text">
           <string>Disable All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reboot_btn">
          <property name="text">
           <string>Reboot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>621</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Engines</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Left Engine</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="launch1">
            <property name="text">
             <string>Launch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop1">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Middle engine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="launch2">
          <property name="text">
           <string>Launch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop2">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Right Engine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="launch3">
          <property name="text">
           <string>Launch</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop3">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Data</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>291</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Data recieving</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="get_temp_btn">
          <property name="text">
           <string>Temperature</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="get_press_btn">
          <property name="text">
           <string>Pressure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="get_alt_btn">
          <property name="text">
           <string>Altitude</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="get_hum_btn">
          <property name="text">
           <string>Humidity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ping_btn">
          <property name="text">
           <string>Ping</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>291</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>Data visualization</string>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>91</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="trackers_upd_time">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lights_upd_time">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="temperature_upd_time">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="humidity_upd_time">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="pressure_upd_time">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="altitude_upd_time">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Update:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Data:</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Trackers:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="trackers_status">
         <property name="text">
          <string>off</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Lights:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lights_status">
         <property name="text">
          <string>off</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Temperature:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="temperature_status">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Humidity:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="humidity_status">
         <property name="text">
          <string>0%</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Air pressure:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="pressure_status">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Flight altitude:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="altitude_status">
         <property name="text">
          <string>0m</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
